tests/from_dict_test.py

```diff
@@ -11,14 +12,16 @@
 from tests.utils import Constructor
 from tests.utils import assert_equal_data
 
-TEST_EAGER_BACKENDS = [
-    Implementation.POLARS,
-    Implementation.PANDAS,
-    Implementation.PYARROW,
-    "polars",
-    "pandas",
-    "pyarrow",
-]
+TEST_EAGER_BACKENDS = []
```

Comment:
Can't remember if variance allows this
```suggestion
TEST_EAGER_BACKENDS: list[Implementation | str] = []
```

Otherwise `list[Any]` would do